A few more improvements to the README.md (#13)
justafish/ddev-drupal-core-dev: README.md

# ddev-drupal-core-dev

* [What is ddev-drupal-core-dev](#what-is-ddev-drupal-core-dev)
* [Initial Setup](#initial-setup)
* [Install Drupal](#install-drupal)

> * [PHP Unit](#install-drupal)

Reviewer: These IDs will need updating to work

> * [Nighwatch](#install-drupal)

Reviewer: Same again here

# ddev-drupal-core-dev

* [What is ddev-drupal-core-dev?](#what-is-ddev-drupal-core-dev)
* [Initial Setup](#initial-setup)
* [Install Drupal](#install-drupal)

> * [PHP Unit](#php-unit)

Reviewer 2: ```suggestion
* [PHPUnit](#phpunit)
```

* [Nighwatch](#nightwatch)

## What is ddev-drupal-core-dev?
This is a [DDEV](https://github.com/ddev/ddev) add-on for doing Drupal Core development.

## Initial Setup

```
git clone https://git.drupalcode.org/project/drupal.git drupal
cd drupal
ddev config --omit-containers=db --disable-settings-management
ddev start

> ddev corepack enable

Reviewer 2: ```suggestion
ddev exec corepack enable
```

> ddev corepack enable

Author: as i've mentioned on discord. `ddev config --omit-containers=db --disable-settings-management` is the exact line used on @simesy's PR. the only overlapping line. therefore i've adjusted it here to avoid conflicts and hassle. and i think those two flags are still needed. for one to avoid the db containers plus to disable the settings management. the rest is done bei config --update

> ddev corepack enable

Author: in regards your corepack suggestion. all the quickstart commands i havent bothered. all of those are taken care of in simes PR. and that corepack line is obsolete and not necessary anymore. config --update takes care of that.

> ddev corepack enable

Reviewer 2: Oh, that PR removes this line? OK. But at least it could be correct. There isn't a `ddev corepack` anywhere. All good.

ddev get justafish/ddev-drupal-core-dev
ddev restart
ddev composer install
````

> Because `ddev-drupal-core-dev` is using SQLite the database container could be omitted on `ddev config --project-type=drupal10 --omit-containers=db`.

Author: In this comment I try to explain "why" the `--omit-container` flag was actually used.

> Because `ddev-drupal-core-dev` is using SQLite the database container could be omitted on `ddev config --project-type=drupal10 --omit-containers=db`.

Reviewer 2: IMO it would be better to do that in a comment above the command. Here it's out of context.

## Install Drupal

To get an overview of the list of available tasks:

```
ddev drupal list
````

Next install a demo site, which is not intended for production:

````
ddev drupal install
````

> > :warning: There is no need to `ddev composer require drush/drush` to get the ability to for example clear caches. A subset of that kind of functionality is available with `ddev drupal list`. Otherwise you won't be able to checkout any added feature branch due to unstaged changes for commit.

Reviewer 2: ```suggestion
> :warning: To avoid unstaged conflicts in git, **do not** `ddev composer require drush/drush`.  A subset of that kind of functionality is available with `ddev drupal list`.
```

> In case you need to drop your database tables because you are for example working in the context of the Drupal installer you can simply manually delete the SQLite database file located in `sites/default/files/.sqlite`.

Reviewer 2: ```suggestion
To drop your database tables, for example if you're working in the context of the Drupal installer, manually delete the SQLite database file located in `sites/default/files/.sqlite`.
```
